dive_core/generatePm4Info_adreno.py

```diff
@@ -826,21 +825,23 @@ def outputPacketInfo(pm4_info_file, registers_et_root, enum_index_dict, opcode_d
       if opcode not in packet_type_instances:
         packet_type_instances[opcode] = 1
         if (opcode == 0):
-          pm4_info_file.write("    // For descriptors, we purposefully try to include them as `packets` for easier parsing.\n")
-          pm4_info_file.write("    // They are not technically PM4 packets, hence the 0x0.\n")
-          pm4_info_file.write("    // Example: const PacketInfo *packet_info_ptr = GetPacketInfo(0, sharp_struct_name);.\n")
+          pm4_info_file.write('''    
+    // For descriptors, we purposefully try to include them as "packets" for easier parsing.
+    // They are not technically PM4 packets, hence the 0x0.
+    // Example: const PacketInfo *packet_info_ptr = GetPacketInfo(0, sharp_struct_name);
+        ''')
```

Comment:
For this multiline string, the whitespace looks intended (it matches the whitespace from before) except for the last line. I believe that the trailing whitespace will be in the output. If this is clang-format'd then maybe this doesn't matter
```suggestion
          pm4_info_file.write('''    
    // For descriptors, we purposefully try to include them as "packets" for easier parsing.
    // They are not technically PM4 packets, hence the 0x0.
    // Example: const PacketInfo *packet_info_ptr = GetPacketInfo(0, sharp_struct_name);
''')
```